apps/examples/testcase/le_tc/drivers/tc_adc.c

```diff
@@ -0,0 +1,114 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_adc.c
+/// @brief Test Case Example for adc driver
+#include <tinyara/config.h>
+#include <tinyara/analog/adc.h>
+#include <tinyara/analog/ioctl.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+#include <errno.h>
+
```

Comment:
can you please add details about below function ?

Reply:
I have not added details because its a very small helper function. And its self explanatory. 
I felt 6 - 8 lines of details would be too much for 3 - 4 line function. 
Please let me know if its necessary.

Reply:
Added one line description